### PR TITLE
Add params to set aria label on inputs

### PR DESCRIPTION
**Pull request by nickbristow**
Allow user to set aria labels for input to make the component accessible.

### CODE REVIEW

**Comment by nickbristow** [comment]
@wojtekmaj 
As noted on the other pr. I made the aria labels props.

File: src/TimePicker.jsx [hunk 1]
```diff
@@ -108,40 +108,51 @@ export default class TimePicker extends PureComponent {
...
           name={name}
+          nativeInputAriaLabel={nativeInputAriaLabel}
           onChange={this.onChange}
           placeholder={this.placeholder}
           required={required}
           value={value}
         />
         {clearIcon !== null && (
           <button
+            aria-label={clearAriaLabel !== null ? clearAriaLabel : undefined}
```
Comment (wojtekmaj) [inline src/TimePicker.jsx]: Why not simply 
```suggestion
            aria-label={clearAriaLabel}
```
? I think `null` is fine and would simply cause `aria-label` not to be added?

File: src/TimePicker.jsx [hunk 2]
```diff
@@ -270,12 +285,15 @@ TimePicker.propTypes = {
   disableClock: PropTypes.bool,
   disabled: PropTypes.bool,
   format: PropTypes.string,
+  hourAriaLabel: PropTypes.string,
   isOpen: PropTypes.bool,
   locale: PropTypes.string,
   maxDetail: PropTypes.oneOf(allViews),
   maxTime: isTime,
+  minAriaLabel: PropTypes.string,
```
Comment (wojtekmaj) [inline src/TimePicker.jsx]: I'd suggest

```suggestion
  minuteAriaLabel: PropTypes.string,
```

since `min` may be confusing. Minute? Minimum?

**Comment by wojtekmaj** [comment]
I'll add some unit tests after you're done, if you don't mind.

**Comment by emilyuhde** [comment]
As part of this PR, it would be really helpful if the table documenting the props in the README file was updated too.

File: src/TimePicker.jsx (hunk 1 above)
Comment (nickbristow) [reply src/TimePicker.jsx]: You're right. I read somewhere it needed to be undefined, but I just tested it and it worked fine.

**Comment by nickbristow** [comment]
> That looks pretty good! Please also add secondAriaLabel :)

@wojtekmaj 
Done, and I've added the params to the readme as well.

**Comment by wojtekmaj** [comment]
@nickbristow Can you enable edits from maintainers? Would love to add some tests before merging.

**Comment by nickbristow** [comment]
@wojtekmaj 
Edits from maintainers is enabled. Sorry for the late response.

**Comment by nickbristow** [comment]
@wojtekmaj 
Any updates? Anything i could help with?

**Comment by wojtekmaj** [comment]
All good! Sorry mate, I'm super busy in my private life. I remember about it!

**Comment by nickbristow** [comment]
@wojtekmaj its all good. Thanks for the update.

**Comment by nickbristow** [comment]
@wojtekmaj Looks like you added some tests. Any chance we can merge? Getting pressure from a client.

**Comment by wojtekmaj** [comment]
@nickbristow This just landed on master, and will be published with the next release. If you're in a hurry, you can already use master branch straight from repo!

**Pull request merged**